app/DoctrineMigrations/Version20181221225017.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types = 1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20181221225017 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE entry CHANGE url url VARCHAR(2048) NOT NULL');
```

Comment:
this was more or less a performance consideration, since this field holds an important unique index and keeping that to a sane size seemed prudent, but since I never experienced any bottlenecks in this area, I guess we can up this

Reply:
It should be increased for using long urls, IE git 2048 and the most other browser more...

Reply:
yes, I agree - please re-read my comment :smile: